invenio_communities/views/communities.py

```diff
@@ -20,6 +20,8 @@
 from jinja2 import TemplateError
 
 from invenio_communities.proxies import current_communities
+from invenio_rdm_records.proxies import current_community_records_service
```

Comment:
major: `invenio-rdm-records` depends on `invenio-communities`.  See [this](https://inveniordm.docs.cern.ch/develop/getting-started/source-code/#repositories-survival-guide) for more information on how the modules are organized (it's a bit outdated)